mayham-game/KinematicBody2D.gd

```diff
@@ -0,0 +1,25 @@
+extends KinematicBody2D
```

Comment:
Why did you create a new KintematicBody2D subclass?

Reply:
I didn't create a new Kinematicbody2D subclass. Is it saying I added the file/lines?

Reply:
That has been removed, so I'm resolving this conversation.